### ISSUE TITLE
GLib-critical

### ISSUE THREAD

**Issue by Panjoo** [jdolan/quetoo#440]
I noticed the following error while alt+tabbing out of the game, which seems to be related to CTF mode. This error message repeats itself whenever a tech is picked up:

![glib](https://cloud.githubusercontent.com/assets/10857572/23276256/eb6e7652-fa09-11e6-9c1d-353e8ff3c46f.jpg)

The first part (quetoo.exe:1600) changes on each session, (ie.: 2688, 5104)

**Comment by jdolan** [comment]
Yea, I think the number is the process ID. @Paril thoughts on this?

**Comment by Paril** [comment]
Bug!

**Comment by WickedShell** [comment]
KaadmY and I had reported this in IRC a couple of times but uh hadn't
opened an issue...

/me hides

**Comment by jdolan** [comment]
This is a server (game) issue, not client (cgame). I just tested this locally with a dedicated server, and the message came through in the server console -- not on the client.

**Comment by Paril** [comment]
I know what the issue is. I'll have it fixed when I am home.

**Comment by Paril** [comment]
So I thought I knew what this issue was but I can't track it down. Somebody that is seeing the error (it's not appearing on Windows) will have to track it down. No calls to has_prefix are even used really, except for one in game that is never called with null.

**Comment by jdolan** [comment]
I'm on the trail, here. I set a breakpoint in `g_log` (Glib's logging facility) and toggled `g_ctf 1`. As soon as my player interacted with a CTF tech, I saw this backtrace:

![screen shot 2017-04-05 at 9 12 41 am](https://cloud.githubusercontent.com/assets/643118/24707021/0a46fe64-19e0-11e7-8163-2d88b22a2591.png)

It looks like the CTF techs do not get initialized with a class name! Yikes! I'm digging into how they are spawned now, but that's an error for sure.
